ui/projects/etl-server/src/lib/pipelines/pipeline-list/pipeline-list.component.html

```diff
@@ -1,6 +1,15 @@
 <app-container>
   <h1 class='theme-primary-fg' i18n>Processing Tasks</h1>
   <div class='controls'>
+    <div *ngIf='roles._.pipelinesNew'>
+      <mat-form-field appearance="fill">
+        <mat-label>Sort</mat-label>
+        <select (change) = 'selectChangeHandler($event)'>
```

Comment:
Use more descriptive name for function -
instead of `selectChangeHandler` you could use for example `sortByChanged`